packages/discord.js/src/structures/Embed.js

```diff
@@ -211,9 +211,30 @@ class Embed {
    */
   equals(other) {
     if (other instanceof Embed) {
-      return isEqual(other.data, this.data);
+      return isEqual(this.data, other.data);
     }
-    return isEqual(other, this.data);
+
+    return (
+      this.color === (other.color ?? null) &&
+      this.description === (other.description ?? null) &&
+      this.title === (other.title ?? null) &&
+      this.url === (other.url ?? null) &&
+      (this.timestamp ? new Date(this.timestamp).getTime() : null) ===
+        (other.timestamp ? new Date(other.timestamp).getTime() : null) &&
+      this.image?.url === other.image?.url &&
+      this.thumbnail?.url === other.thumbnail?.url &&
+      this.video?.url === other.video?.url &&
```

Comment:
All of these are missing comparisons on height, width, and proxy URL

Reply:
I've been using the code snippet from Jiralite's issue but with different embeds and those properties are never present when using `<Message>.embeds[0].data` so that's why I didn't add these comparisons